pages/admin/articles.js

```diff
@@ -28,7 +28,7 @@ const mapStateToProps = (state, { lang }) => ({
 const ARTICLE_COLS = [
   {
     id: 'title',
-    className: b('font', { size: 'large' }),
+    className: styles['wir-table__font--size--large'],
```

Comment:
left looks better)

Reply:
it includes `__font` class though which breaks styling. should be refactored (one day)